cg/meta/workflow/mip_dna.py

```diff
@@ -0,0 +1,363 @@
+# -*- coding: utf-8 -*-
+import gzip
+import logging
+import re
+from pathlib import Path
+from typing import List, Any
+from ruamel.yaml import safe_load
+
+from requests.exceptions import HTTPError
+
+from cg.apps import tb, hk, scoutapi, lims
+from cg.meta.deliver.mip_dna import DeliverAPI
+from cg.apps.pipelines.fastqhandler import BaseFastqHandler
+from cg.store import models, Store
+
+
+COLLABORATORS = ("cust000", "cust002", "cust003", "cust004", "cust042")
+MASTER_LIST = (
+    "ENDO",
+    "EP",
+    "IEM",
+    "IBMFS",
+    "mtDNA",
+    "MIT",
+    "PEDHEP",
+    "OMIM-AUTO",
+    "PIDCAD",
+    "PID",
+    "SKD",
+    "NMD",
+    "CTD",
+    "IF",
+    "NEURODEG",
+    "mcarta",
+)
+COMBOS = {
+    "DSD": ("DSD", "HYP", "SEXDIF", "SEXDET"),
+    "CM": ("CNM", "CM"),
+    "Horsel": ("Horsel", "141217", "141201"),
+}
+CAPTUREKIT_MAP = {
+    "Agilent Sureselect CRE": "agilent_sureselect_cre.v1",
+    "SureSelect CRE": "agilent_sureselect_cre.v1",
+    "Agilent Sureselect V5": "agilent_sureselect.v5",
+    "SureSelect Focused Exome": "agilent_sureselect_focusedexome.v1",
+    "Twist Human core exome v1.3 + Twist Human RefSeq Panel": "Twist_Target_hg19_RefSeq.bed",
+    "other": "agilent_sureselect_cre.v1",
+}
+
+
+class AnalysisAPI:
```

Comment:
This AnalysisAPI is so similar to the mip_dna AnalysisAPI. Could the shared code be generalised into a common parent class?

Reply:
We want to have it like this to start with to decouple the pipelines, might be that we generalize this later